lib/node_modules/@stdlib/blas/base/sscal/include/sscal.h

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
Move this file to `include/stdlib/blas/base/*`. See [`ddot`](https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/blas/base/ddot).